docs/dashboard-setup/whitelisting.mdx

```diff
@@ -0,0 +1,87 @@
+displayed_sidebar: docs
+description: "Client ID and Whitelisting references | Documentation - Web3Auth"
+---
+
+import DashboardImage from "@site/static/dashboard/project_plug_n_play.png";
+
+### Whitelisting URLs using the Developer Dashboard
+
+The Whitelist URL is the address you wish to authorize for your project. When implementing Web3Auth, it is important to whitelist the specific URLs
+from which you intend to make calls for user authentication. By doing so, you can ensure that these activities are conducted securely and efficiently.
+Remember to whitelist all relevant URLs to guarantee seamless integration with Web3Auth.
+
+Type in the URL you wish to whitelist in the text box and click on the `Add` button. The URL will be added to the list of whitelisted URLs. You can
+add as many URLs as you want.
+
+<div style={{ flexBasis: "300px", flexGrow: "1", textAlign: "center" }}>
+  <img src={DashboardImage} style={{ alignSelf: "center", maxWidth: "100%" }} alt="Dashboard Whitelist" />
+</div>
+
+### Whitelisting URLs using the OpenloginAdapter
+
+You can use the manual origin whitelisting method to whitelist URLs without using the Developer Dashboard. `OpenLoginAdapter` accepts a parameter
+called `originData` inside the `adapterSettings`. `originData` is a key value pair where key is the origin URL and the value is a `signature`. The
+signature is generated by `whitelistUrl` function from the `@toruslabs/openlogin` package. The `whitelistUrl` function accepts the `clientId`,
+`clientSecret` and `origin` as parameters.
+
+```js
+import { OpenloginAdapter } from "@web3auth/openlogin-adapter";
+import { whitelistUrl } from "@toruslabs/openlogin";
+import { CommonPrivateKeyProvider } from "@web3auth/base-provider";
+
+const clientId = "YOUR_CLIENT_ID"; // get from https://dashboard.web3auth.io
+const clientSecret = "CORRESPONDING_CLIENT_SECRET"; // get from https://dashboard.web3auth.io
+const origin = "https://example.com";
+
+const privateKeyProvider = new CommonPrivateKeyProvider({ config: { chainConfig } });
+const sig = await whitelistUrl(clientId, clientSecret, origin);
+
+const openloginAdapter = new OpenloginAdapter({
+  privateKeyProvider,
+  adapterSettings: {
+    originData: { origin: sig },
+  },
+});
+```
+
+A snippet from the SDK reference for `originData` is given below:
+
+```ts
+    /**
+     * originData is used to verify the origin of dapp by iframe.
+     *
+     * @internal
+     * @remarks
+     * You don't have to pass originData explicitly if you have registered your dapp at
+     * {@link "https://dashboard.web3auth.io"| developer dashboard}.
+     *
+     * originData contains a signature of dapp's origin url which is generated using
+     * project's secret.
+     */
+    originData?: OriginData;
+```
+
+### Whitelisting for Mobile apps
+
+You can use the Developer Dashboard to whitelist URLs for mobile apps as well. Or you can try out the the programmatic whitelisting method to whitelist URLs.
```

Comment:
Don't recommend programmatic way. Add more context on what is deeplinking and what deeplinking URLs need to be generated. Add links to the SDK Reference page where deeplinking is mentioned for each platform in PnP SDKs.